src/Product/ProductList.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from "react";
+import Product from "./Product";
+
+const API_URL = "https://px32id5fdg.execute-api.us-east-1.amazonaws.com/data";
+
+function ProductList() {
+  const [productData, setProductData] = useState([]);
+  const [searchInput, setSearchInput] = useState("");
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    fetch(API_URL)
```

Comment:
This is a small nitpick but I think it's a good habit to call `setLoading(true)` before you fetch. For this application, you don't need to do that because the default value is already true and the app will only fetch on the initial load. Even so, it's a good habit to use this pattern and explicitly set loading to true before any asynchronous action like fetching data from an API.